frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -105,12 +105,11 @@ common_settings.network = {
 }
 NetworkMgr:getMenuTable(common_settings)
 
+basicSettings = 1
```

Comment:
Nitpick, but our style guide has `basic_settings` the variable and `basicSettings()` the function so you can tell at a glance what's going on. :-)

However, I have a more fundamental comment, see below.